GrandCentralBoardTests/APIDataProviderTests.swift

```diff
@@ -35,7 +35,10 @@ class TestRequestManager : NetworkRequestManager {
         self.token = token
     }
 
-    func requestJSON(method: GrandCentralBoard.Method, url: NSURL, parameters: [String : AnyObject]? = nil, headers: [String : String]? = nil, completion: (ResultType<AnyObject, NSError>.result) -> Void) {
+    func requestJSON(method: GrandCentralBoard.Method, url: NSURL, parameters: [String : AnyObject]? = nil,
```

Comment:
maybe drop `parameters` to new line too so all parameters are in separate lines for consistency?